packages/protocol/scripts/foundry/create_and_migrate_anvil_devchain.sh

```diff
@@ -60,6 +64,39 @@ $NON_INTERACTIVE \
 $LIBRARY_FLAGS \
 --rpc-url $ANVIL_RPC_URL || echo "Migration script failed"
 
+# Set the supply for Celo:
+# This supply is just an approximation, but it's importat that this number is non-zero.
+
+# Get CeloToken contract address
+CELO_ADDRESS=$(
+  cast call \
+  $REGISTRY_ADDRESS \
+  "getAddressForStringOrDie(string calldata identifier)(address)" \
+  "CeloToken" \
+  --rpc-url $ANVIL_RPC_URL
+)
+
+# interpersonate the VM, this can't be done from the migration script
+cast rpc anvil_impersonateAccount $CELO_VM_ADDRESS --rpc-url $ANVIL_RPC_URL
+
+# set the balance of the vm address so that it can send a tx
+cast rpc anvil_setBalance $CELO_VM_ADDRESS $ETHER_IN_WEI --rpc-url $ANVIL_RPC_URL
+
+# increase the supply (it is harcoded as bash overflows)
+# ideally this number should come from the amount of address funded when Anvil loads
```

Comment:
I couldn't figure out how to tell anvil how many addresses to fund, so hardcoded the 9

Reply:
I think this is a known limitation. I looked into this in the past and couldn't find an easy way to access the number of default accounts: 
- https://github.com/foundry-rs/foundry/issues/5196